app/javascript/dashboard/routes/dashboard/commands/conversationHotKeys.js

```diff
@@ -349,7 +374,12 @@ export default {
       return [];
     },
   },
-
+  mounted() {
```

Comment:
@scmmishra Is there any better way to do this show/hide functionality? We need to support it dynamically https://github.com/chatwoot/chatwoot/pull/9361/files#r1592712599

Reply:
Sure, let me take a look

Reply:
Updated the PR with the suggestions